### PR TITLE
[QPPA-1362] Update QCDR measures with changes from PIMMS

### PR DESCRIPTION
**Pull request by kalvinwang**
This PR updates the QCDR measures with changes since 10/25.

- [x] Manually update the updated PIMMS QCDR spreadsheet to include multiPerformanceRate strata changes from QPPA-1310
- [x] Incorporate the 164 changes to the PIMMS QCDR spreadsheet since 10/25
- [x] Define strata names for the four new measures which are newly proportional/multiPerformanceRate measures (MICS1, MICS2, MICS3, RCOIR1)
- [x] Send email to ask for future updates to include multiPerformanceRate strata changes

Also:
- I'm removing the old PIMMS QCDR spreadsheets; I think it makes more sense to keep updating qcdr-measures.csv with new changes, so that diffs are useful. 
- The qcdr-measures.csv has been uploaded to the jira ticket as `2017 PIMMS QCDR measures v4`

Jira: https://jira.cms.gov/browse/QPPA-1362

Tests: Still passes all tests

Reviewer: @kencheeto

### CODE REVIEW

File: util/measures/qcdr-measures-strata-names.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "ARCO13": ["antithrombotic", "anticoagulation", "antithrombotic2", "LDL"],
   "AHSQC9": ["overall", "email"],
   "AQI49": ["lysine", "mini", "redcell", "transfusion", "overall"],
-  "MOA1": ["back", "neck"],
+  "MOA 1": ["back", "neck"],
```
Comment (kencheeto) [inline util/measures/qcdr-measures-strata-names.json]: @lamroger is it an issue to have a space in this measure id?

Comment (kalvinwang) [reply util/measures/qcdr-measures-strata-names.json]: (FYI, the space comes from the source data, it's not added on our end. I removed the space last time manually, but decided against re-removing it this time to make future changes less manual.)

**Comment by kalvinwang** [comment]
@kencheeto I don't know of any overlap between the qcdr csv and benchmarks... were you thinking of anything in particular?

File: util/measures/qcdr-measures-strata-names.json (hunk 1 above)
Comment (kencheeto) [reply util/measures/qcdr-measures-strata-names.json]: i think @lamroger removed the spaces when he parsed a csv elsewhere, so let's do that (remove whitespace)

Comment (kalvinwang) [reply util/measures/qcdr-measures-strata-names.json]: hm ok... done!

**Comment by kalvinwang** [comment]
@kencheeto ohhh right, ok. nope, no corresponding benchmarks update.

**Pull request merged**